lib/racecar/runner.rb

```diff
@@ -2,12 +2,17 @@
 
 module Racecar
   class Runner
-    attr_reader :processor, :config, :logger
+    attr_reader :processor, :config, :logger, :consumer
 
     def initialize(processor, config:, logger:)
       @processor, @config, @logger = processor, config, logger
     end
 
+    def stop
+      processor.stop if processor.respond_to?(:stop)
```

Comment:
I needed this so much, couldn't fine a better name than `stop`, `teardown` is so much more obvious. :)